tests/extproc/extproc_test.go

```diff
@@ -143,55 +157,7 @@ func TestE2E(t *testing.T) {
 		}, 10*time.Second, 1*time.Second)
 	})
 
-	t.Run("streaming", func(t *testing.T) {
```

Comment:
do not delete the existing test....

Reply:
Probably due to the bad rebase, it was just added.